installation/phonesetup.md

```diff
@@ -0,0 +1,125 @@
+
+Your phone needs to have an unlocked bootloader and be able to support applications running as root. Its recommended that you install [Lineage OS](https://lineageos.org/).  They have good installation instructions for each device.  Make sure to install the Playstore as described in the Lineage instructions. If you are an expert you can use a different OS, if not use Lineage.
+>Always check the exact version of the phone, some version does not work with some ROMs. For ex. Samsung has numerous versions numbers of the S5. Some are able to install Lineage and others are not. Lineage lists the phone version numbers that it is compatible with.
+
+>Samsung S5 G900V and G900A are confirmed NOT to work. G900T DOES work.
+Moto G4 works, but the Amazon edition of the Moto G4 (Moto G4 Play) does NOT work.
+
+Once you have Lineage installed and the Google Apps with the Playstore, move on to Magisk.
+
+## Magisk
+
+1. Install [Magisk](https://www.xda-developers.com/how-to-install-magisk/) to root the phone via recovery. Download it [here](https://github.com/topjohnwu/Magisk/releases)
+     >There is currently a bug in the very latest Magisk version that prevents RGC from clicking on the screen (which is needed for questscanning), use Magisk 19.0 or lower to avoid this.
+
+2. Repackage the MagiskManager App and make sure to delete the folder `/sdcard/MagiskManager` after repackaging. If its not present, you are good to go.
+
+## Smali Patcher
+
+The Smali Patcher is a little program written in Java to patch a android file called `services.jar`. That will allow you to use the "mock location" feature from Android.
+
+1. Follow the instructions here to download and install it from the [Smali Patcher Forum](https://forum.xda-developers.com/apps/magisk/module-smali-patcher-0-7-t3680053)
+2. You can ether install it via TWRP or via the magisk module option in the magisk manager.
+3. Reboot to TWRP to delete the Davlik and the normal Cache.
+
+## Applications - PoGo, RGC & PogoDroid
+
+### Pokémon Go
+
+Install the PoGo app from the Play Store and add PoGo to Magisk Hide via the magisk manager.
+
+>It's necessary to pass the Safetynet check to run the game on rooted phones. Check the Safetynet status in the MagiskManager App.
+
+### RGC
+
+RGC - Install [RGC (Remote GPS Controller)](https://github.com/Map-A-Droid/MAD/blob/master/APK/RemoteGpsController.apk)
+
+- RGC must be converted to a system app after it is installed.  There are two ways to do this, link2sd and Systemizer Magisk Module.  Start with Link2sd and then go to Systemizer if it doesnt work.
+
+- Install [Link2sd](https://play.google.com/store/apps/details?id=com.buak.Link2SD) to systemize RGC. Once installed go into Link2sd, you may be prompted to give it super user, if so do it.  Scroll down and find RGC.  Click on it, go to the menu and convert RGC to a system app.  It may ask you to reboot, if so do so.
```

Comment:
```suggestion
- Install [Link2sd](https://play.google.com/store/apps/details?id=com.buak.Link2SD) to systemize RGC. Once installed go into Link2sd, you may be prompted to give it superuser access, if so do accept this.  Scroll down and find RGC.  Click on it, go to the menu and convert RGC to a system app.  It may ask you to reboot if so do so.
```